pkg/webhook/validate_test.go

```diff
@@ -164,6 +164,40 @@ func TestValidateSriovOperatorConfigWithDefaultOperatorConfig(t *testing.T) {
 	g.Expect(ok).To(Equal(true))
 }
 
+func TestValidateSriovNetworkAllMulticast(t *testing.T) {
+	var err error
+	var ok bool
+
+	network := &SriovNetwork{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: "test",
+		},
+		Spec: SriovNetworkSpec{
+			AllMulticast: "on",
+			Trust:        "on",
+		},
+	}
+
+	g := NewGomegaWithT(t)
+	ok, _, err = validateSriovNetwork(network, "CREATE")
+	g.Expect(err).NotTo(HaveOccurred())
+	g.Expect(ok).To(Equal(true))
+
+	ok, _, err = validateSriovNetwork(network, "UPDATE")
+	g.Expect(err).NotTo(HaveOccurred())
+	g.Expect(ok).To(Equal(true))
+
+	network.Spec.Trust = "off"
```

Comment:
nit: please have a separate test for the off so it's cleaner thanks :)

Reply:
sure, thanks

Reply:
done